### PR TITLE
Documentation Ilan

### PR DESCRIPTION
**Pull request by ilanefraimovic**
Added Documentation for backend, included color component for players to see which is the current color in the game. This lets the next player know which color he has to play, as well as helps people who may have certain visual impairments.

### CODE REVIEW

File: frontend/src/components/Town/interactables/UnoTable.tsx
Comment (SamuelLev24) [inline frontend/src/components/Town/interactables/UnoTable.tsx:1]: Good display fixes.

File: frontend/src/components/Town/interactables/UnoTable.tsx [hunk 1]
```diff
@@ -62,20 +54,20 @@ const unoTable: React.FC<UnoTableProps & { interactableID: InteractableID }> = (
...
 
   const calculatePlayerPosition = (index: number, ourPlayerIndex: number, totalPlayers: number) => {
     const relativeIndex = ((index - ourPlayerIndex + totalPlayers) % totalPlayers);
     const angle = (2 * Math.PI) / totalPlayers;
-    const radius = 20; // Adjust this value as needed
-    const x = 42.5 + radius * 1.5 * Math.sin(angle * relativeIndex);
-    const y = 55 + radius * Math.cos(angle * relativeIndex);
+    const radius = 25; // Adjust this value as needed
+    const x = 45 + radius * 1.5 * Math.sin(angle * relativeIndex);
+    const y = 32.5 + radius * Math.cos(angle * relativeIndex);
```
Comment (SamuelLev24) [inline frontend/src/components/Town/interactables/UnoTable.tsx:64]: May be better to use constants for future changes if necessary

File: townService/src/town/games/UnoGame.ts
Comment (SamuelLev24) [inline townService/src/town/games/UnoGame.ts:1]: Good documentation.

**Review by samlev24 — APPROVED** [review]
Everything looks good to me and can be merged into main! I'll go ahead and do so while fixing merge conflicts from previous pr.

**Comment by SamuelLev24** [comment]
I approved changes just a bit ago, and have now resolved all merge conflicts that were present. I will now merge this pr.

**Pull request merged**